### ISSUE TITLE
Children support

### ISSUE THREAD

**Issue by Noitidart** [Jpoliachik/react-native-triangle#9]
Would be awesome for children support!

**Comment by Jpoliachik** [comment]
I don't think this approach will work for rendering children, since we're rendering the view with width=0, height=0, and using the border to draw the triangle.
